src/legacy/core_plugins/kibana/common/field_formats/types/url.js

```diff
@@ -157,7 +157,9 @@ export function createUrlFormat(FieldFormat) {
             }
             // Handle urls like: `../app/kibana`
             else {
-              prefix = `${parsedUrl.origin}${parsedUrl.basePath}/app/`;
+              const prefixEnd = url[0] === '/' ? '' : '/';
+
+              prefix = `${parsedUrl.origin}${parsedUrl.basePath || ''}/app${prefixEnd}`;
```

Comment:
Could you please add some tests for broken case

Reply:
Sure. Added